Makefile

```diff
@@ -119,7 +119,7 @@ clean: clean_docs
 	rm -rf $(BUILD_DIR)
 	rm -f $(GOPATH)/bin/crc
 	rm -rf $(RELEASE_DIR)
-       
```

Comment:
Modifications that are grouped under unrelated issue. Please create a ne wcommit

Reply:
Removed Makefile from this commit

Reply:
Thanks